protocol/flatbuffers/rpc.fbs

```diff
@@ -94,6 +96,11 @@ table FilteringSettings {
     ticks: uint8;
 }
 
+table FkSettings{
+    toggles: [FkSkeletonToggle];
```

Comment:
Why are these lists? What does the index of the list mean? This should be explained in a comment since its not clear.

Reply:
The lists contains all the toggles and values. The index would point to which toggle or which value.

Reply:
Added a comment